src/app/pages/About/About.html

```diff
@@ -0,0 +1,18 @@
+<div class="about">
+  <h1>{{ $t('app.title') }}</h1>
+
+  <h2>Contributors</h2>
+
+  <div class="row justify-content-center">
+    <div class="col-lg-2 col-md-4 col-sm-6 col-xs-12">
```

Comment:
Personally I prefer to sort these breakpoints starting with smallest `col-xs-*` because you should start with mobile first.

Reply:
This will be refactored in a later stage.